File: hard/5_sorted_squares/sorted_squares_test.go
package main

import (
	"github.com/stretchr/testify/require"
	"testing"
)

func TestSortedSquares(t *testing.T) {
	input := []int{1, 2, 3, 5, 6, 8, 9}
	expected := []int{1, 4, 9, 25, 36, 64, 81}
	actual := SortedSquaredArray(input)
	require.Equal(t, expected, actual)
}

File: hard/5_sorted_squares/sorted_squares_test.py
import unittest
from sorted_squares import sortedSquaredArray

class TestSortedSquares(unittest.TestCase):
	def test_sorted_squares(self):
		input = [1, 2, 3, 5, 6, 8, 9]
		expected = [1, 4, 9, 25, 36, 64, 81]
		actual = sortedSquaredArray(input)
		self.assertEqual(expected, actual)
